docs/install-upgrade/config.md

```diff
@@ -208,10 +90,89 @@ spec:
   bootstrap:
     disk: "/dev/sda" # disk to install OS on, e.g. "sda" or "nvme0n1"
   external:
-    interface: eno2 # interface for external
+    interface: eno2 # customer interface to manage control node
     ip: dhcp # IP address for external interface
-  management:
+  management: # interface that manages switches in private management network
     interface: eno1
 
 # Currently only one ControlNode is supported
 ```
+
+### Configure Control Node and Switch Users
+
+#### Control Node Users
+Configuring control node and switch users is done either passing 
+`--default-password-hash` to `hhfab init` or editing the resulting `fab.yaml` 
+file emitted by `hhfab init`.  The default username on the control node is
+core. 
+
+#### Switch Users
+There are two users on the switches, `admin` and `operator`. The `operator` user has
+read-only access to `sonic-cli` command on the switches. The `admin` user has
+broad administrative power on the switch. 
+In order to avoid conflicts, do not use the following usernames: `operator`,`hhagent`,`netops`.
+
+### NTP and DHCP
+The control node uses public ntp servers from cloudflare and google by default.
+The control node runs a dhcp server on the management network. See the [example file](#complete-example-file).
```

Comment:
> [example file](#complete-example-file)

The subsection name has changed, this link is broken. Is there no way to insert reference targets and link to them with Mkdocs? :scream_cat:

Reply:
I did some looking into this. Thanks to [this](https://stackoverflow.com/questions/6695439/how-to-link-to-a-named-anchor-in-multimarkdown) Stack Overflow link. We can create a reference, by adding an anchor tag. but markdown already makes anchor tags from headings, so it would be redundant in most cases. One simply needs to remember to update the the links....and check the docs build output which indicates when links are broken. It is best to have the build output *not* lost in the Z ordering of the desktop.

Reply:
Oh good if it can detect broken links at build, do you know if there's a chance to make it error in that case so we can catch it in CI?